JWBestPracticeApps/FeedTableViewController/FeedTableViewController.swift

```diff
@@ -11,6 +11,7 @@ import UIKit
 class FeedTableViewController: UITableViewController {
     
     var feed = [JWPlayerController]()
+    var thumbnailsIdentifiers = [String]()
```

Comment:
Instead of an array, I'd use a dictionary, and allow it to be looked up by title or url. This way not all players need to have a thumbnail image, especially since through you logic below it is possible to not have feed.count == thumbnailsIdentifiers.count

Reply:
I have made this modification in the next commit

Reply:
Looking more into this, I don't believe you need this array at all. Since you are only saving the url of the image as a String, this is already done in the JWConfig, which you will have access to though 'feed' (JWPlayerController.config). This way you don't have data duplicated, and don't chance 'feed' and 'thumbnailsIdentifiers' being arrays of different sizes.

Reply:
I'm not using the url to get the image, instead I'm using the localThumbnail to get the name of the asset of image to load on the UIImage